File: android/QLCHDT_API/QLCHDT_API/Models/Nhacungcap.cs
using System;
using System.Collections.Generic;

namespace QLCHDT_API.Models
{
    public partial class Nhacungcap
    {
        public Nhacungcap()
        {
            Sanpham = new HashSet<Sanpham>();
        }

        public string Mancc { get; set; }
        public string Tenncc { get; set; }
        public string Diachi { get; set; }
        public string Dienthoai { get; set; }
        public string Chuthich { get; set; }

        public ICollection<Sanpham> Sanpham { get; set; }
    }
}

File: android/DoAnMonHocAndroid/DoAnMonHocAndroid/app/src/main/java/com/example/doanmonhoc/NhaCungCap/NhaCungCap.java
package com.example.doanmonhoc.NhaCungCap;

import java.io.Serializable;

public class NhaCungCap implements Serializable {
    String mancc, tenncc, diachi,dienthoai,chuthich;

    public NhaCungCap() {
    }

    public NhaCungCap(String mancc, String tenncc, String diachi, String dienthoai, String chuthich) {
        this.mancc = mancc;
        this.tenncc = tenncc;
        this.diachi = diachi;
        this.dienthoai = dienthoai;
        this.chuthich = chuthich;
    }

    public String getMancc() {
        return mancc;
    }

    public void setMancc(String mancc) {
        this.mancc = mancc;
    }

    public String getTenncc() {
        return tenncc;
    }

    public void setTenncc(String tenncc) {
        this.tenncc = tenncc;
    }

    public String getDiachi() {
        return diachi;
    }

    public void setDiachi(String diachi) {
        this.diachi = diachi;
    }

    public String getDienthoai() {
        return dienthoai;
    }

    public void setDienthoai(String dienthoai) {
        this.dienthoai = dienthoai;
    }

    public String getChuthich() {
        return chuthich;
    }

    public void setChuthich(String chuthich) {
        this.chuthich = chuthich;
    }
}
